packages/web-components/package.json

```diff
@@ -90,6 +90,7 @@
     "test:dev": "playwright test"
   },
   "devDependencies": {
+    "@axe-core/playwright": "^4.10.0",
```

Comment:
please move this to root `package.json` devDeps in order to adhere to [single version policy](https://github.com/microsoft/fluentui/issues/21395) ty.

Reply:
Moved, thanks!